CONTRIBUTING.md

```diff
@@ -145,10 +145,14 @@ To save @bors some work, and to get small changes through more quickly, when
 the other rollup-eligible patches too, and they'll get tested and merged at
 the same time.
 
-To find documentation-related issues, sort by the [A-docs label][adocs]. 
+To find documentation-related issues, sort by the [A-docs label][adocs].
 
 [adocs]: https://github.com/rust-lang/rust/issues?q=is%3Aopen+is%3Aissue+label%3AA-docs
 
+To render documentation you can use rustdoc. For example, `rustdoc
+src/doc/reference.md` will render reference to `doc/reference.html`. The css
+might be messed up, but you can verify that html is right.
```

Comment:
Thanks for this!

The English is great. :smile: A few things: 

* `rustdoc` should be in graves, so it renders like code.
* CSS and HTML not css and html
* Can you add something to indicate that this might not always work? Something like

> In many cases, you don't need a full `make doc`. You can use `rustdoc` directly to check small fixes. For example...